renderer/index.js

```diff
@@ -318,7 +320,7 @@ function saveTorrentData (torrent) {
 function startTorrenting (torrentId) {
   var torrent = state.client.add(torrentId, {
     // use downloads folder
-    path: state.saved.downloads
+    path: state.saved.downloadPath
```

Comment:
I think I know what that issue is caused by. Will post a comment about it in a sec.